lib/extensions.js

```diff
@@ -41,6 +41,7 @@ function getHumanArchitecture(arch) {
     case 'ia32': return '32-bit';
     case 'x86': return '32-bit';
     case 'x64': return '64-bit';
+    case 'ppc64': return '64-bit';
```

Comment:
@nschonni - below is contents of lscpu on my ppc64le system
```
[root@p006vm11 ~]# lscpu
Architecture:          ppc64le
Byte Order:            Little Endian
CPU(s):                16
On-line CPU(s) list:   0-15
Thread(s) per core:    1
Core(s) per socket:    1
Socket(s):             16
NUMA node(s):          1
Model:                 IBM pSeries (emulated by qemu)
L1d cache:             64K
L1i cache:             32K
NUMA node0 CPU(s):     0-15

```
However what node expect it as below
```
[jenkins@463995b0acbe Amex_Node_Sass_Rhel_7.2]$ node -p "process.arch"
ppc64
```

So that's the reason made above change